app/assets/stylesheets/govuk_publishing_components/components/print/_step-by-step-nav.scss

```diff
@@ -59,6 +58,9 @@ $stroke-width: 3px;
   background: $white;
   border-radius: 100px;
   text-align: center;
+  font-size: 19px;
```

Comment:
i guess we're not using the font mixin as we want consistent font-size across breakpoint?

Reply:
Note that by doing so, when compatibility mode is disabled this text won't be resizable (as unlike everything that uses the typography mixin, it won't be defined in rem). Might be something to worry about in the future though.

Reply:
Correct, this is to ensure a consistent font size across breakpoints. I'll add a comment about the compatibility mode, as that's a good point, but something to worry about later.